cypress/fixtures/capabilities.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"provider_type":"remote","package_url":"https://github.com/layer5labs/meshery-extensions-packages/releases/download/v0.6.68-8/provider-meshery.tar.gz","provider_name":"Meshery","provider_description":["Persistent sessions","Save environment configurations and user preferences","Store and retrieve patterns, filters, and application defintions","Retrieve performance and conformance test results","RBAC support","Free use"],"provider_url":"https://meshery.layer5.io","extensions":{"navigator":[{"title":"MeshMap","on_click_callback":1,"href":{"uri":"/meshmap","external":false},"component":"/provider/navigator/meshmap/index.js","icon":"/provider/navigator/img/meshmap-icon.svg","link":true,"show":true,"type":"full_page"}],"user_prefs":[{"component":"/provider/userpref/meshmap_userpref/index.js","type":"component"}],"graphql":[{"component":"meshmap","path":"provider/navigator/meshmap/graphql/plugin.so","type":"backend"}],"account":[{"title":"Account","on_click_callback":1,"href":{"uri":"/account","external":false},"component":"/provider/account/profile/index.js","link":true,"show":true,"type":"full_page"}]},"capabilities":[{"feature":"sync-prefs","endpoint":"/user/preferences"},{"feature":"persist-results","endpoint":"/results"},{"feature":"persist-result","endpoint":"/result"},{"feature":"persist-smi-results","endpoint":"/smi/results"},{"feature":"persist-smi-result","endpoint":"/smi/result"},{"feature":"persist-metrics","endpoint":"/result/metrics"},{"feature":"persist-smp-test-profile","endpoint":"/user/test-config"},{"feature":"persist-performance-profiles","endpoint":"/user/performance/profiles"},{"feature":"persist-schedules","endpoint":"/user/schedules"},{"feature":"persist-meshery-patterns","endpoint":"/patterns"},{"feature":"persist-meshery-filters","endpoint":"/filters"},{"feature":"persist-meshery-applications","endpoint":"/applications"},{"feature":"persist-meshery-pattern-resources","endpoint":"/patterns/resource"}],"package_version":"v0.6.68-8"}
```

Comment:
It auto-modifies fixtures when the releaseTag is configured via [setReleaseTag](https://github.com/layer5labs/meshery-extensions-packages/blob/778bc6a57e2f9fe21797acc7ca8bfd0fc6b526c9/cypress/support/commands.js#L35-L48) command.

It doesn't have any relevance for any change, thus, I think I should add it under `.gitignore` if it won't be subject to future change.

Reply:
I think its bad idea to put it under `.gitignore`, since at later point the changes made may be relevant to be pushed. I will not push the changes made in this unless required from future.